rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2025,4 +2026,27 @@ void testGetUTXOsSentToAddresses_no_utxo_sent_to_given_address_after_RSKIP293()
 
         Assertions.assertTrue(foundUTXOs.isEmpty());
     }
+
+    @Test
```

Comment:
Should these tests go in BridgeSerializationUtilsTest?

Reply:
yes, my bad. Thanks